resources/migrations/001_update_migrations.yaml

```diff
@@ -4390,6 +4390,52 @@ databaseChangeLog:
                     nullable: true
                   remarks: 'If true, the table requires a filter to be able to query it'
 
+  - changeSet:
+      id: v49.00-013
+      author: johnswanson
+      comment: Add `api_key.updated_by_id`
+      rollback: # will be removed with the table
+      changes:
+        - addColumn:
+            tableName: api_key
+            columns:
+              - column:
+                  remarks: The ID of the user that last updated this API key
+                  name: updated_by_id
+                  type: integer
+                  constraints:
+                    # no default and nullable: false for a new column? yikes! But this should probably be ok, because
+                    # we don't yet have any UI for creating a new API key.
+                    nullable: false
+                    referencedTableName: core_user
+                    referencedColumnNames: id
+                    foreignKeyName: fk_api_key_updated_by_id_user_id
+
+  - changeSet:
+      id: v49.00-014
+      author: johnswanson
+      comment: Add an index on `api_key.updated_by_id`
+      rollback: # not necessary, will be removed with the table
+      changes:
+        - createIndex:
+            tableName: api_key
+            indexName: idx_api_key_updated_by_id
+            columns:
+              column:
+                name: updated_by_id
+
+  - changeSet:
+      id: v49.00-015
+      author: johnswanson
+      comment: Rename `created_by` to `created_by_id`
```

Comment:
We actually have `creator_id` as the column name in some tables, like `report_dashboard`. Maybe use that instead for consistency? 

You could also call the other one `updater_id` but it sounds a little less natural

Reply:
I thought about this before and was torn between:
- `created_by_id` is consistent with `updated_by_id` in the same table, but inconsistent with other tables
- `creator_id` is consistent with other tables, but then not consistent with `updated_by_id`
But I think you're right and I'll change this - the inconsistency between `creator_id` and `updated_by_id` is pretty minimal. And if we wanted to hydrate the `creator` of the ApiKey, we could do so already (since it's already set up), while we'd need another line of code to tell t2 how to automagically hydrate the `created_by`.